source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -24,13 +24,15 @@ information in the context of a real-world scenario:
 - **How to implement Client-Side Field Level Encryption with the MongoDB
   driver** (`Implementation`_)
 
-For a complete runnable example project, see the `Additional Information`_
-section.
+.. note::
```

Comment:
This section was changed into a `note` based on user research that showed it was not visible enough:
This section was changed in response to feedback provided by

Reply:
nit: Maybe this could be a note admonition titled **Download the Code**?

Reply:
Sounds good to me, will update.